### PR TITLE
user/v-danwesley/writer pass on "Get started viewing and changing CSS"

### PR DESCRIPTION
**Pull request by dan-wesley**

### CODE REVIEW

**Review by dan-wesley — COMMENTED** [review]
Line 47 The `Inspect Me!` element is highlighted in the Styles panel. <-- **It isn't**
Line 163  **which tip?**
line 167 not sure where the device toolbar is and don't know if the previous screenshot is good
line 179 not sure why screenshot shows 2 url when selecting media query. Maybe need to use a different webpage example?

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/css/index.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Use the **Box Model** interactive diagram in the **Styles** panel to change the
 
    Depending on the size of your DevTools window, you may need to scroll to the bottom of the **Styles** panel to display the **Box Model**.
 
-1. Double-click the left margin in the **Box Model**, which currently has a value of `-`, meaning that the element doesn't have a `margin-left`.
+1. Double-click the left margin in the **Box Model**, which currently has a value of `-`. The `-` means that the element doesn't have a value for `margin-left`.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/css/index.md:137]: add red boxes on the next couple screenshots

File: microsoft-edge/devtools-guide-chromium/css/index.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ To debug or test the Media Queries you defined in your CSS:
 
    ![Selecting a media query from the preview bar.](../media/css-elements-media-queries-select-bar.msft.png)
 
-1. To debug media queries and open the CSS file in the `Sources` editor, right-click a bar segment and then select `reveal in source code`.
+1. To debug media queries and open the CSS file in the `Sources` editor, right-click a bar segment, and then select `reveal in source code`.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/css/index.md:177]:
```suggestion
1. To debug media queries and open the CSS file in the `Sources` editor, right-click a bar segment, and then select **Reveal in source code**.
```

File: microsoft-edge/devtools-guide-chromium/css/index.md [hunk 3]
```diff
@@ -150,13 +150,13 @@ Use the **Box Model** interactive diagram in the **Styles** panel to change the
...
-Using separate layouts allows for a one-column layout for mobile devices and multi-column layouts when there is more screen estate available.
+Using separate layouts allows for a one-column layout for mobile devices and multi-column layouts when there's more screen real estate available.
 
-To debug or test the Media Queries you defined in your CSS:
+Use the following steps to debug or test the Media Queries you defined in your CSS.
 
-1. Open DevTools.  To do this, you can right-click in a webpage and then select **Inspect**.
+1. To open DevTools, you can right-click in a webpage and then select **Inspect**.
 
 1. Click the **Toggle device emulation** (![Device Emulation icon.](../media/device-emulation-icon-light-theme.png)) button.  Or, when DevTools has focus, press `Ctrl`+`Shift`+`M` (`Cmd`+`Shift`+`M` on macOS).
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/css/index.md:161]:
```suggestion
1. Click the **Toggle device emulation** (![Device Emulation icon.](../media/device-emulation-icon-light-theme.png)) button.  Or, when DevTools has focus, press `Ctrl`+`Shift`+`M` (Windows, Linux) or `Cmd`+`Shift`+`M` (macOS).
```

File: microsoft-edge/devtools-guide-chromium/css/index.md [hunk 4]
```diff
@@ -22,55 +22,55 @@ ms.date: 05/04/2021
...
 
-1. Open the [CSS Examples](https://microsoftedge.github.io/Demos/devtools-css-get-started/) demo page in a new window or tab.  To do this, right-click the link, or press and hold `Ctrl` (Windows, Linux) or `Command` (macOS) and then click the link.
+1. Open the [CSS Examples](https://microsoftedge.github.io/Demos/devtools-css-get-started/) demo page in a new window or tab.  To open the examples, right-click the link, or press and hold `Ctrl` (for Windows, Linux) or `Command` (for macOS) and then click the link.
 
    <!-- You can view the source files for the CSS Examples demo page at the [MicrosoftEdge/Demos > devtools-css-get-started](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-css-get-started) repo folder. -->
 
 1. Right-click the `Inspect Me!` text, and then select **Inspect**.
 
-   In DevTools, on the **Elements** tool, in the **DOM Tree** panel, the `Inspect Me!` element is highlighted.
+   In the **DOM Tree** panel under the **Elements tool**, you’ll see that the `Inspect Me!` element is highlighted.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/css/index.md:37]:
```suggestion
   In the **DOM Tree** panel under the **Elements tool**, the `Inspect Me!` element is highlighted.
```

File: microsoft-edge/devtools-guide-chromium/css/index.md (hunk 1 above)
Comment (dan-wesley) [reply microsoft-edge/devtools-guide-chromium/css/index.md:137]: added red boxes, should be visible now

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**